B/Birch_Standard/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Birch_Standard"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/lawmurray/Birch.git", "a533990b59b04ef4cd20f8d4fe91c4a220df1cfa"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+# add libraries for building birch on host
+apk add boost-dev yaml-dev
+
+# build and install driver on host system
+mkdir -p ${WORKSPACE}/srcdir/build_host
+autoreconf -vi
```

Comment:
```
[16:30:17]  ---> autoreconf -vi
[16:30:17] autoreconf: 'configure.ac' or 'configure.in' is required
```

Reply:
I lost a line somehow when I copied it from the previous PR.